sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * [Request and Dependency Success criteria will now be decided based on
   `Activity.Status`](https://github.com/Azure/azure-sdk-for-net/pull/31024)
 * [Changed `AzureMonitorTraceExporter` to internal](https://github.com/Azure/azure-sdk-for-net/pull/31067)
+* Changed offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor".
```

Comment:
1. Add link to the actual PR.
2. What does this mean to the reader/user? (what does the user need to do with it?)

Reply:
1. Thanks for the catch! Updating this now.
2. Meant to be informative, if a user was expecting files in one directory that will have changed. I don't think it's actionable.

Reply:
I felt this was justified for the changelog. The local storage is something we have to disclose because data here is unencrypted. [link](https://docs.microsoft.com/en-us/azure/azure-monitor/app/data-retention-privacy#does-the-sdk-create-temporary-local-storage)

Reply:
> I felt this was justified for the changelog. The local storage is something we have to disclose because data here is unencrypted. [link](https://docs.microsoft.com/en-us/azure/azure-monitor/app/data-retention-privacy#does-the-sdk-create-temporary-local-storage)

+1

Reply:
> 2. Meant to be informative, if a user was expecting files in one directory that will have changed. I don't think it's actionable.

Does the user have an option to explicitly specify a path/folder?

Reply:
Yes! `AzureMonitorExporterOptions.StorageDirectory`
Should I call attention to this in the changelog?

Reply:
Consider writing changelog in a way that makes sense to the reader. For example, instead of "updated the ABS system logic", use "adjusted the brake system to provide better handling when the road is wet/slippery, this feature will be automatically turned off when the vehicle is at 25mph or less speed".

Here are the key points:
1. What is the actual change.
2. Why should the user care about it, what is the side-effect for the user.
3. What should the user do if they don't want to be affected, or there is nothing they can do besides taking the impact.